python/cudf/cudf/tests/test_hdfs.py

```diff
@@ -69,6 +69,18 @@ def test_csv(tmpdir, pdf, hdfs, test_url):
     assert_eq(expect, got)
 
 
+@pytest.mark.parametrize("test_url", [False, True])
+def test_write_csv(pdf, test_url):
+    gdf = cudf.from_pandas(pdf)
+    if test_url:
+        hd_fpath = "hdfs://{}:{}{}/file.csv".format(host, port, basedir)
+    else:
+        hd_fpath = "hdfs://{}/file.csv".format(basedir)
+
+    with pytest.raises(Exception):
```

Comment:
Can you make this more specific instead of a general Exception catch?

Reply:
It's currently a `RuntimeError`. Is that okay?

Reply:
Yes, lets just have it catch that instead of all exceptions here to make sure we're not overshadowing something else breaking :)

Reply:
Updated to catch more specific error